### DISCUSSION TITLE
How to remove the trojan within this .dll of the SDR++ frequency_manager !?

### DISCUSSION THREAD

**Discussion by KavorkaDesigns** [AlexandreRouma/SDRPlusPlus#1505] [Q&A]
How to remove the **trojan** within this .dll?? SDR++ frequency_manager
@AlexandreRouma 
[GitHubLink](https://github.com/AlexandreRouma/SDRPlusPlus/tree/master/misc_modules/frequency_manager)
VirusTotal results
[VT Link](https://www.virustotal.com/gui/file/1fe4064e8f386191838f79624b251c0b01175255682b59d85d5b84608d8d5672/detection)
![VirS](https://github.com/user-attachments/assets/25d91c62-e1c7-42cc-8f38-d8c5a007f6bc)

**Comment by AlexandreRouma** [comment]
there is no trojan whatsoever, it's some antivirus being stupid

**Comment by KubaPro010** [comment]
its literally open source, there is no trojan

**Comment by KavorkaDesigns** [comment]
The identified markers from VT's results are within the file itself, I just open the file with HxD and NP++. Open or Closed, the markers are in there. I'll find another app..

![BewareOfInfectedSDRSoftware](https://github.com/user-attachments/assets/397429e1-af92-4085-bbe0-ca256147f405)

**Reply by KubaPro010** [reply]
you can literally read the source code and compile it yourself, there's no closed or open

**Reply by KubaPro010** [reply]
also, it's literally a single antivirus which is known for false positives (https://github.com/restic/restic/issues/4430) and is crappy in the first place (https://www.reddit.com/r/antivirus/comments/rz7gzn/maxsecure_the_antivirus_i_dont_recommend/)

(i hate these "cybersecurity specialists" when they call something a virus when a single indian antivirus detected it and even is not sure (it says `susgen`) and that app is *open-source*)

**Reply by AlexandreRouma** [reply]
the "markers" you're showing are literally the handler functions every SDR++ module has to define.
![image](https://github.com/user-attachments/assets/8f646bb9-c4a0-4dea-8a0d-534680e3671c)

**Comment by KavorkaDesigns** [comment]
Just being cautious. The device is new and I have much studying to do yet. I scanned the files and seen this, I agree the AV isn't even known haha. I'll have to freshen up on my coding, I'll update if I make anything good. I'll be removing this post shortly, thanks for your input
